OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -599,13 +608,17 @@ void testBushingForce()
 
     osimModel.setGravity(gravity_vec);
 
-    BushingForce spring("bushing", "ground", "ball",
+    auto* spring = new BushingForce("bushing", "ground", "ball",
         transStiffness, rotStiffness, transDamping, rotDamping);
 
-    osimModel.addForce(&spring);
+    osimModel.addForce(spring);
     const BushingForce& bushingForce =
         osimModel.getComponent<BushingForce>("bushing");
 
+    // It's necessary to correct the connectee names in the BushingForce, which
+    // we can do with finalizeConnections() (they are incorrect otherwise
+    // because `spring` is initially orphaned).
+    osimModel.finalizeConnections(osimModel);
```

Comment:
Can you update the comment to something like: "To print (serialize) the latest connections of the model, it is necessary to finalizeConnections() first." Or something to that effect? I think the code is fine.